nix-plugins/include/flox/core/util.hh

```diff
@@ -0,0 +1,277 @@
+/* ========================================================================== *
```

Comment:
nonblocking: do we still need everything in here? Looks like you deleted the tests

Reply:
Apparently not, strippted the vec equality and nlohman extensions. left logs and the overload template.